src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -360,6 +361,15 @@
     FLUSH_LOCAL_SCHEMA_CHANGES("cassandra.test.flush_local_schema_changes", "true"),
 
     TOMBSTONE_HISTOGRAM_TTL_ROUND_SECONDS("cassandra.streaminghistogram.roundseconds", "60"),
+
+
+    /**
+     * {@link }}StorageCompatibility mode sets how the node will behave, sstable or messaging versions to use etc according to a yaml setting. 
+     * But many tests don't load the config hence we need to force it otherwise they would run always under the default.
+     *
+     * This is a dev/CI only property. Do not use otherwise.
```

Comment:
Nit: maybe we can repeat here the nice explanation that there is in `DatabaseDescriptor.getStorageCompatibilityMode`

Reply:
Done

Reply:
I don't see any changes here, maybe you forgot to merge/push?

Reply:
Don't you see that comment in the preceding paragraph?

```
...Config is null for junits that don't load the config. Get from env var that CI/build.xml sets.
```

Reply:
Aaah, right, it's on the above paragraph! Yes, that's exactly what I meant. Sorry I've missed it, I left the comment on the second paragraph and didn't see the change on the first one after the rebase. +1.